src/include/sof/audio/module_adapter/intel_module_adapter/module_initial_settings_concrete.h

```diff
@@ -0,0 +1,131 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+
+#ifndef _MODULE_INITIAL_SETTINGS_CONCRETE_H
+#define _MODULE_INITIAL_SETTINGS_CONCRETE_H
+
+
+#include "module_initial_settings.h"
+#include <utilities/array.h>
+#include <sof/compiler_attributes.h>
+#include <ipc4/base-config.h>
+
+#include <stddef.h>
+
+
+typedef struct ipc4_base_module_cfg BaseModuleCfg;
+
```

Comment:
this struct name should be changed to something that has no reference to specific ipc protocol version

Reply:
The ipc4_base_module_cfg is type of structure passed during init instance IPC. It is used across other build in modules in SOF, like aria, up_down_mixer, volume and src. IADK modules works only with IPC4 protocol. Why not to use it here?
I can define BaseModuleCfg type directly, but it will exact match the ipc4_base_module_cfg definition.